pipelines/README.md

```diff
@@ -0,0 +1,23 @@
+# Pipeline Definitions
+
+This directory contains pipeline definition YAML files which are used
+to verify calibration products with the LSST Science Pipelines.
+The pipelines defined here are science ready and come in two flavors:
+generic ("ingredients") and camera-specific (within appropriately
+named sub-directories).  Use of camera-specific pipelines is strongly
+encouraged where possible as they are optimized for the particular
+characteristics of that camera.
+
+The pipelines defined here tend to import other pipelines, including
+ingredient pipelines in the [_ingredients](./_ingredients) directory.
```

Comment:
This can just be `[_ingredients](_ingredients)` I think. I'd also recommend one line per sentence in this file too.